### PR TITLE
x86_64: Add support for EFI framebuffer (FS#515)

### PR DESCRIPTION
**Pull request by alive4ever**
CONFIG_FB_EFI and CONFIG_FB_SIMPLE are enabled for x86_64 to display
console text on EFI framebuffer.

Tested on Qemu with EDK2 2.60.

Signed-off-by: alive4ever <alive4ever@live.com>

### CODE REVIEW

File: target/linux/x86/64/config-default [hunk 1]
```diff
@@ -226,7 +227,6 @@ CONFIG_PARAVIRT=y
 CONFIG_PARAVIRT_CLOCK=y
 # CONFIG_PARAVIRT_DEBUG is not set
 CONFIG_PARAVIRT_SPINLOCKS=y
-# CONFIG_PARAVIRT_TIME_ACCOUNTING is not set
 CONFIG_PATA_AMD=y
```
Comment (blogic) [inline target/linux/x86/64/config-default]: this change is unrelated

**Comment by blogic** [comment]
The Signed-off-by line needs to be a full real name and no nickname please

**Comment by alive4ever** [comment]
Ok. I'll file a new pull request on another branch.
